### ISSUE TITLE
Generated code is not compliant with noUnusedParameters

### ISSUE THREAD

**Issue by Hexxeh** [creditkarma/thrift-typescript#116]
`struct MyResponse {
}`

generates the following code:

`export class MyResponse {
    constructor(args?: IMyResponseArgs) {
    }
}`

which with noUnusedParameters causes errors

**Comment by kevin-greene-ck** [comment]
Yeah, we don't document that it will work with that flag, just `noUnusedLocals`. I would like to add this support. Not sure when we will have the bandwidth.
